server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ConditionDataName.java

```diff
@@ -56,6 +60,15 @@ public void addConditionName(ConditionDataName variable) {
 
   @Override
   public Variable rename(RenameItem newParent) {
-    return new ConditionDataName(name, definition, newParent, value);
+    return new ConditionDataName(name, definition, newParent, value, valueTo);
+  }
+
+  @Override
+  public String getFormattedDisplayLine() {
+    StringBuilder stringBuilder = new StringBuilder();
+    stringBuilder.append(String.format("%1$02d %2$s VALUE %3$s", levelNumber, name, value));
+    if (valueTo != null)
```

Comment:
Are you allow ifs without curly brackets in che4z codebase?

Reply:
Yes, I clarified it when I introduced the style check system.
And personally, I like it when curly brackets are not mandatory. So I was not arguing a lot)